index.js

```diff
@@ -209,6 +209,9 @@ export const Classes = {
 		userAward: 'user-award',
 		userAwards: 'user-awards'
 	},
+	condition: {
```

Comment:
Should this be plural `conditions`? I think we tend to use this as a grouping of the classes, so classes related to `grades`, classes related to `awards`, classes related to `conditions`, etc.

Reply:
Also matches the namespace, `conditions.api.brightspace.com` :)